Horvat Diana Adriana/lab 5 pb 3,5/main.c

```diff
@@ -0,0 +1,113 @@
+#include <stdio.h>
+#include <stdlib.h>
+int n;
+void prod(int **x, int **y, int **z, int n)
+{
+    for(int i=0; i<n; i++)
+    {
+        for(int j=0; j<n; j++)
+        {
+            x[i][j]=0;
+
+            for(int k=0; k<n; k++)
+            {
+                x[i][j]=x[i][j]+(y[i][k]*z[k][j]);
+            }
+        }
+    }
+
+}
+void sum(int **x, int **y, int **z, int n)
+{
+    for(int i=0; i<n; i++)
+    {
+        for(int j=0; j<n; j++)
+        {
+            x[i][j]=(y[i][j]+z[i][j])*2;
+        }
+    }
+
+}
+void sub(int **x, int **y, int **z, int n)
+{
+    for(int i=0; i<n; i++)
+    {
+        for(int j=0; j<n; j++)
+        {
+            x[i][j]=y[i][j]-z[i][j];
+        }
+    }
+
+}
+int **a,**b,**c,**aux1,**aux2;
+int main()
+{
+    scanf("%d",&n);
+
+    int **aux2=(int**)malloc(sizeof(int*)*n);
```

Comment:
This is fine, but seeing as every matrix is allocated the same amount of space (`n`) why not just condense everything in 1 or 2 for loops?